activemodel/test/cases/validations/numericality_validation_test.rb

```diff
@@ -338,7 +345,6 @@ def test_validates_numericality_with_invalid_args
     assert_raise(ArgumentError) { Topic.validates_numericality_of :approved, greater_than: "foo" }
     assert_raise(ArgumentError) { Topic.validates_numericality_of :approved, less_than: "foo" }
     assert_raise(ArgumentError) { Topic.validates_numericality_of :approved, equal_to: "foo" }
-    assert_raise(ArgumentError) { Topic.validates_numericality_of :approved, in: "foo" }
```

Comment:
I'm not sure about removing this.  I understand why -- `String` responds to `include?` -- but I wonder if we should specifically check for `String` and raise an `ArgumentError`.  Or perhaps check `value.respond_to?(:include?) && !value.is_a?(Enumerable)`.  (This would negate my previous suggestion involving `CLUSIVITY_CHECKS.any? { ... }`.)